lib/containers/agda/Haskell/Data/Set.agda

```diff
@@ -0,0 +1,130 @@
+{-# OPTIONS --erasure #-}
+
+-- | Postulates and definitions of the operations supported by 'Set'.
+module Haskell.Data.Set where
+
+open import Haskell.Prelude hiding (lookup; null; map; filter)
+
+{-----------------------------------------------------------------------------
+    Data.Set
+------------------------------------------------------------------------------}
+
+-- | As the term 'Set' is already taken in Agda, we use ℙ (\bP).
+postulate
+  ℙ : Set → Set
```

Comment:
Nasty clash with powerset notation; why not require the `--no-import-sorts` flag and use `Set` as expected?